recipes-domu/domu-image-fusion/files/meta-xt-prod-extra/recipes-extended/netconfig/files/eth.network

```diff
@@ -0,0 +1,5 @@
+[Match]
```

Comment:
How did it work before without netconfig?

Reply:
removed